zaplings/templates/zaplings/creatorsnight.html

```diff
@@ -162,7 +162,8 @@ <h5 class="inline-title add-margin-top"></h5><h4 class="inline-title add-margin-
                 </div>
             </div>
 
-            <form id="rsvp-basic" name="rsvp-basic">
+           <form id="rsvp-basic" name="rsvp-basic" action="{% url 'zaplings:process-rsvp' %}" method="post"/>
```

Comment:
@dannypernik You've overwritten my previous change that turned this form into an actionable one. This is also the reason why you were not seeing the user being redirected to rsvp confirm upon submitting.

Here is the link with your change:
https://github.com/permalife/zaplings-full/commit/8098113f815793ff1a435d3dc45288497818b090#diff-484ffc701f1af5a20a0886a833c91ace